To: vim_dev@googlegroups.com
Subject: Patch 7.3.244
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.244
Problem:    MS-Windows: Build problem with old compiler. (John Beckett)
Solution:   Only use HandleToLong() when available. (Mike Williams)
Files:	    src/gui_w32.c


*** ../vim-7.3.243/src/gui_w32.c	2011-05-25 21:18:02.000000000 +0200
--- src/gui_w32.c	2011-07-07 17:42:36.000000000 +0200
***************
*** 1574,1579 ****
--- 1574,1583 ----
  #endif
  
  #ifdef FEAT_EVAL
+ # if _MSC_VER < 1400
+ /* HandleToLong() only exists in compilers that can do 64 bit builds */
+ #  define HandleToLong(h) ((long)(h))
+ # endif
      /* set the v:windowid variable */
      set_vim_var_nr(VV_WINDOWID, HandleToLong(s_hwnd));
  #endif
*** ../vim-7.3.243/src/version.c	2011-07-07 17:36:52.000000000 +0200
--- src/version.c	2011-07-07 17:43:21.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     244,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
262. Your computer has it's own phone line - but your daughter doesn't.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
